src/model/Scenario.js

```diff
@@ -100,7 +100,7 @@ var Scenario = new Class( /** @lends Scenario# */ {
 					break;
 				case 'string':
 				case 'number':
-					winston.loggers.get('load').debug('Oops, encoutered "' + sourceStep + '" as a free step in a scenario!'	// TODO: remove this hint after v0.4
+					winston.loggers.get('load').debug('Oops, encoutered "' + sourceStep + '" as a free step!'	// TODO: remove this hint after v0.4
```

Comment:
Because it cost nothing to keep it, had no negative impact, and the `number` type was not used for anything else  ;)